### PR TITLE
Routerchanges

### PR DESCRIPTION
**Pull request by DylanDKim**
***Changes to react router in next pull request -
Adam helped me get the event id into the url params so now you can access it by
import { useParams } from ‘react-router-dom’;
const { event_id: eventId } = useParams();
and use it for your axios requests, to access the sample data, sample data is at
61b79b9e0ac02dbe3e12fd1b
ex) http://localhost:3000/event/61b79b9e0ac02dbe3e12fd1b/dashboard
 axios.get(`https://betrothed-server.herokuapp.com/api/events/${eventId}`)
.then(data => console.log(data))
I added /event/:event_id to the router so after my pull request you’ll have to use that format to reach your page, any number works for :event_id if you’re not serving data to it from the db
ex) http://localhost:3000/event/1/dashboard
for your buttons that link to other pages will have to follow this format as well
ex)
import { Link, useParams } from ‘react-router-dom’;
const { event_id: eventId } = useParams();
const link = `/event/${eventId}/guestlist`;
then in JSX:
<Link to={link} class=“text-decoration-none”>
<Button>Guestlist</GuestList>
</Link>

### CODE REVIEW

File: src/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,36 +1,98 @@
...
+import { BFaustinaH2 } from '../Common/styled/textstyles/FaustinaH2';
 
 export default function Dashboard(props) {
   const [firstName, setFirstName] = useState('John');
   const [partnerFirstName, setPartnerFirstName] = useState('Jane');
   const [daysLeft, setDaysLeft] = useState('2');
+  const [progress, setProgress] = useState(0);
+
+  const [numGuests, setNumGuests] = useState(0);
+  const [numGifts, setNumGifts] = useState('');
```
Comment (kyle-sablan) [inline src/Dashboard/Dashboard.js:20]: would numGifts default state also be a number state like numGuests? just a minor comment that doesn't seem to affect any flow or anything

File: src/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -1,36 +1,98 @@
...
-      <Row
+      <h1>event_id is: {eventId}</h1>
+      {/* <Row
         className="d-flex justify-content-between"
-        style={{ marginTop: '1em' }}
+        style={{ marginTop: '3em' }}
       >
-        <Col>
+        <Col md={{ offset: 1 }}>
           <BAlexBrushH1>
```
Comment (kyle-sablan) [inline src/Dashboard/Dashboard.js:80]: could this comment be removed?

File: src/Onboarding/Onboarding.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import { Outlet } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios';
+
+export default function OnboardingBase(props) {
+  const [email, setEmail] = useState('email.addres');
+
+  console.log(email);
+
+  const body = {
```
Comment (kyle-sablan) [inline src/Onboarding/Onboarding.js:11]: is this body variable used?

**Review by kyle-sablan — APPROVED** [review]
Just some minor comments but overall everything is looking really solid!

File: src/Dashboard/Dashboard.js [hunk 4]
```diff
@@ -1,36 +1,98 @@
...
             {firstName} and {partnerFirstName}
           </BAlexBrushH1>
         </Col>
         <Col className="d-flex justify-content-end">
           <BFaustinaH3>{daysLeft} days left!</BFaustinaH3>
         </Col>
-      </Row>
-      <div style={{ height: '16em', backgroundColor: 'grey' }}>
+      </Row> */}
```
Comment (hithisiskhai) [inline src/Dashboard/Dashboard.js:87]: This looks like half a comment that didn't get deleted

**Review by hithisiskhai — APPROVED** [review]

**Pull request merged**